tasks/pythonz.yml

```diff
@@ -24,7 +24,7 @@
     force: no
     repo: https://github.com/saghul/pythonz.git
     dest: /usr/src/pythonz
-    version: c3fcfa8876eda7fcafe55ae244d6884a4d540aeb
+    version: master
```

Comment:
Should we keep the version pinned? I just don’t love assuming a branch isn’t the in-flight and could break on us

Reply:
Same but looking at pythonz commit history it seems pretty stable and only change to support new python versions (checksum & autocomplete). 

IMO it should be ok. I could make it a default variable so people could choose

Reply:
Works for me!